src/Umbraco.Web.UI.Client/src/views/components/tabs/umb-tabs-nav.html

```diff
@@ -1,5 +1,5 @@
-<ul class="umb-tabs-nav">
-    <li class="umb-tab" ng-repeat="tab in vm.tabs |  limitTo: vm.maxTabs" data-element="tab-{{tab.alias}}" ng-class="{'umb-tab--active': tab.active, 'umb-tab--error': tabHasError}" val-tab>
+<ul role="tablist" class="umb-tabs-nav">
+    <li class="umb-tab" role="tab" aria-selected="true" aria-controls="panel-1" id="tab-1" tabindex="0" ng-repeat="tab in vm.tabs |  limitTo: vm.maxTabs" data-element="tab-{{tab.alias}}" ng-class="{'umb-tab--active': tab.active, 'umb-tab--error': tabHasError}" val-tab>
```

Comment:
@tiffy74 it seems to be invalid, where all tabs get the id `tab-1`. You could change it to `id="tab-{{$index + 1}}"` and maybe `tabindex="{{$index}}"`.

Reply:
Thanks for the response

Actually I had intended it to be simpler - I just forgot to remove the additional elements from my own testing code.  The id="tab-1" was meant to be removed.

It should be:

`<li class="umb-tab" role="tab" aria-selected="true" tabindex="0" ng-repeat="tab in vm.tabs |  limitTo: vm.maxTabs" data-element="tab-{{tab.alias}}" ng-class="{'umb-tab--active': tab.active, 'umb-tab--error': tabHasError}" val-tab>
        <a ng-href="" ng-click="vm.clickTab($event, tab)">{{ tab.label }}</a>
    </li>`

Reply:
Have now updated the branch and removed the id and aria-controls elements - tested and works as expected.